docs/navigation/overlay/component-context.md

```diff
@@ -0,0 +1,69 @@
+# How to initialize Child Overlay with custom ComponentContext 
+
+In order to pass this `AppComponentContext` to child overlay components, make an extension function on the `AppComponentContext` interface. This custom extension function will initialize the `Child Overlay` and provide child `AppComponentContext`.
+
+```kotlin
+fun <C : Parcelable, T : Any> AppComponentContext.appChildOverlay(
```

Comment:
This sample looks very similar to `appChildStack`. Do you find dedicated samples actually helpful? What do you think if we just write something like the following after `appChildStack` samples?

> Creating `Child Overlay` with custom `ComponentContext` is very similar to `Child Stack`. Please refer to the samples above.

Reply:
I did some changes here, please check. I will be a bit redundant but...